metricbeat/metricbeat.reference.yml

```diff
@@ -156,6 +156,18 @@ metricbeat.modules:
   # Password of hosts. Empty by default
   #password: password
 
+#-------------------------------- Beats Module -------------------------------
+- module: beats
```

Comment:
Shouldn't this be the `beat` module? You monitor just 1 beat at the time.

Reply:
Sure, I will rename.

Reply:
Done.